src/hotspot/share/opto/callnode.cpp

```diff
@@ -489,6 +489,7 @@ void JVMState::format(PhaseRegAlloc *regalloc, const Node *n, outputStream* st)
         ciField* cifield;
         if (iklass != NULL) {
           st->print(" [");
+          iklass->nof_nonstatic_fields(); // FIXME: iklass->_nonstatic_fields == NULL
```

Comment:
It was intended as a quick hack and `JVMState::format()` is not the right place to solve the problem.
I suggest to look at `PhaseVector::scalarize_vbox_node`. 

Here are some pointers on how it is done in other places:
  https://github.com/openjdk/jdk/blob/master/src/hotspot/share/opto/macro.cpp#L740
  https://github.com/openjdk/jdk/blob/master/src/hotspot/share/opto/memnode.cpp#L1320